client/src/components/ResultListEntry.jsx

```diff
@@ -4,6 +4,15 @@ import { Router, Route, Link, IndexRoute, hashHistory, browserHistory, DefaultRo
 class ResultListEntry extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      favColleges: ""
+    }
+    this.addToFav = this.addToFav.bind(this);
+  }
+
+  addToFav(e) {
+     console.log(this.props.college);
+     this.props.handleFav(this.props.college);

```

Comment:
This will break when this entry is being rendered for the favorites because it won't be able to properly reference handleFav.